src/components/common/PostalCode/PostalCodeInput.tsx

```diff
@@ -1,26 +1,62 @@
-import { useStorage } from '@faststore/sdk'
+import { useSession, useStorage } from '@faststore/sdk'
 import { Input as UIInput, Label as UILabel } from '@faststore/ui'
 import React, { useRef } from 'react'
 import type { KeyboardEvent } from 'react'
+import { gql } from '@vtex/graphql-utils'
+import { request } from 'src/sdk/graphql/request'
+import type {
+  RegionQueryQuery,
+  RegionQueryQueryVariables,
+} from '@generated/graphql'
 
 import './postal-code-input.scss'
 
 const POSTAL_CODE_STORAGE_KEY = 'main::store::postalCode'
 const POSTAL_CODE_INPUT_ID = 'postal-code-input'
 
+export const RegionQuery = gql`
+  query RegionQuery($session: IStoreSession!) {
```

Comment:
Region? We shouldn't call it region

Reply:
Yeah, the objective of this query is fetch the region and nothing more.